telegram/ext/filters.py

```diff
@@ -372,1895 +363,1807 @@ def name(self, name: str) -> NoReturn:
 
-        class _Edited(UpdateFilter):
-            __slots__ = ()
-            name = 'Filters.update.edited'
+        def filter(self, update: Update) -> bool:
+            return update.edited_channel_post is not None
 
-            def filter(self, update: Update) -> bool:
-                return update.edited_message is not None or update.edited_channel_post is not None
+    EDITED_CHANNEL_POST = _EditedChannelPost()
+    """Updates with :attr:`telegram.Update.edited_channel_post`."""
 
-        edited = _Edited()
+    class _Edited(UpdateFilter):
+        __slots__ = ()
+        name = 'filters.UpdateType.EDITED'
 
-        class _ChannelPosts(UpdateFilter):
-            __slots__ = ()
-            name = 'Filters.update.channel_posts'
+        def filter(self, update: Update) -> bool:
+            return update.edited_message is not None or update.edited_channel_post is not None
 
-            def filter(self, update: Update) -> bool:
-                return update.channel_post is not None or update.edited_channel_post is not None
+    EDITED = _Edited()
+    """Updates with either :attr:`telegram.Update.edited_message` or
+    :attr:`telegram.Update.edited_channel_post`."""
 
-        channel_posts = _ChannelPosts()
+    class _ChannelPosts(UpdateFilter):
+        __slots__ = ()
+        name = 'filters.UpdateType.CHANNEL_POSTS'
 
         def filter(self, update: Update) -> bool:
-            return bool(self.messages(update) or self.channel_posts(update))
+            return update.channel_post is not None or update.edited_channel_post is not None
 
-    update = _UpdateType()
-    """Subset for filtering the type of update.
+    CHANNEL_POSTS = _ChannelPosts()
+    """Updates with either :attr:`telegram.Update.channel_post` or
+    :attr:`telegram.Update.edited_channel_post`."""
 
-    Examples:
-        Use these filters like: ``Filters.update.message`` or
-        ``Filters.update.channel_posts`` etc. Or use just ``Filters.update`` for all
-        types.
+    def filter(self, update: Update) -> bool:
+        return bool(self.MESSAGES.check_update(update) or self.CHANNEL_POSTS.check_update(update))
 
-    Attributes:
-        message: Updates with :attr:`telegram.Update.message`
-        edited_message: Updates with :attr:`telegram.Update.edited_message`
-        messages: Updates with either :attr:`telegram.Update.message` or
-            :attr:`telegram.Update.edited_message`
-        channel_post: Updates with :attr:`telegram.Update.channel_post`
-        edited_channel_post: Updates with
-            :attr:`telegram.Update.edited_channel_post`
-        channel_posts: Updates with either :attr:`telegram.Update.channel_post` or
-            :attr:`telegram.Update.edited_channel_post`
-        edited: Updates with either :attr:`telegram.Update.edited_message` or
-            :attr:`telegram.Update.edited_channel_post`
-    """
+
+UPDATE = UpdateType()
+"""Shortcut for :class:`telegram.ext.filters.UpdateType()`."""
```

Comment:
This does basically the same as `filters.ALL`, right? What would you think of just dropping it and making `UpdateType` just a convenience namespace as `ChatType`?

If you don't like dropping, I'd again vote for `filters.UpdateType.ALL` :)

Reply:
a convenience namespace it is